docs/continuous_integration.rst

```diff
@@ -1,11 +1,17 @@
 Continuous Integration
 ======================
 
-CumulusCI Flow
---------------
+.. toctree::
+   :maxdepth: 2
+
+   cumulusci_flow.rst
+   github_actions.rst
+
 
-Run CumulusCI from Github Actions
----------------------------------
 
 MetaCI
 ------
+The Salesforce.org release engineering team, which built CumulusCI, also maintains a CI system called `MetaCI <https://github.com/SFDO-Tooling/MetaCI>`_.
+MetaCI is an open source app built to run on Heroku, and is designed specifically to work with CumulusCI and Salesforce.
+However, MetaCI is a bit complicated to set up and operate, so this recipe aims to provide a simpler alternative that can work fine in many cases.
```

Comment:
This is no longer on the same page as "this recipe" (which was the info about github actions)